int32 diffPackets
int32 diffBytes
int32 deltaFramesDropped
int32 deltaPackets
int32 deltaBytes
